docs/IntegrationTest.md

```diff
@@ -13,31 +13,6 @@
 -->
 # Integration Tests
 
-## Jenkins on GKE with helm
-Follow Google Cloud Solutions'
-[tutorial](https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial) for setting up 
-Jenkins on GKE.
-
-<!--
-TODO(stephenashank): Add another step for the CasC config when we are able to pre-load credentials
-through the helm chart after refactoring the Google Oauth Plugin.
--->
-Before running `helm install`, add the following modification to the file `jenkins/values.yaml`:
-
-* Directly below the line which reads `Agent:`, add the additional lines shown here:
-    ```yaml
-    Agent:
-        Image: gcr.io/jenkins-gke-plugin/jenkinsagent
-        ImageTag: latest
-    ```
-
-See the [Dockerfile](../jenkinsagent/Dockerfile) for the `gcr.io/jenkins-gke-plugin/jenkinsagent`
-image.
-
-When running `helm install` remove the flag `--version` and its argument.
```

Comment:
I wanted to avoid dictating to the user what version of the chart to use, as they may have need of a specific version, (if they chose to have an ingress for example). Our plugin doesn't depend upon a specific version of the chart, so I thought it better to leave it up to the user.

Reply:
Then can we make a note about the fact that the tutorial does specify a version in the helm install (which is very outdated), and maybe ask for changes in the tutorial which remove the version flag, which shouldn't be needed on their end either.